examples/src/main/java/com/mapbox/navigation/examples/core/camera/MapboxCameraAnimationsActivity.kt

```diff
@@ -183,8 +183,8 @@ class MapboxCameraAnimationsActivity :
                 )
             }
 
-            viewportDataSource.evaluate()
             if (locationMatcherResult.isTeleport) {
+                viewportDataSource.evaluate()
```

Comment:
This cuts the number of calls in half. I'm not seeing a reason why we need to evaluate route progress and the matched location updates.

Reply:
Considering that a different approach may need to be considered for high frequency locations.